caiman/utils/visualization.py

```diff
@@ -1153,22 +1153,37 @@ def nb_inspect_correlation_pnr(corr, pnr):
 
         pnr: ndarray
             peak-to-noise image created with caiman.summary_images.correlation_pnr
+
```

Comment:
Might be better to type-annotate these in the prototype rather than give them types in the comment, and document the return function there too